components/editor/modules/list/index.js

```diff
@@ -105,7 +109,7 @@ export default ({rule, subModules, TYPE}) => {
           const isEmpty = !inItem || !inItem.text
 
           if (isEmpty && (!isBackspace || inList.nodes.size === 1)) {
-            return change.unwrapBlock()
+            return change.setNodeByKey(inList.key, { type: 'paragraph' })
```

Comment:
we shouldn't hard code type – need to be re-configured able through schema if we want to render things differently. Here we transition to a p without list right? Why not unwrap?